internal/gengapic/testdata/method_GetBigThing.want

```diff
@@ -1,5 +1,6 @@
 func (c *FooClient) GetBigThing(ctx context.Context, req *mypackagepb.InputType, opts ...gax.CallOption) (*GetBigThingOperation, error) {
-	ctx = insertMetadata(ctx, c.xGoogMetadata)
+	md := metadata.Pairs("x-goog-request-params", fmt.Sprintf("%s=%v", "field_name", req.GetFieldName()), "x-goog-request-params", fmt.Sprintf("%s=%v", "other", req.GetOther()))
```

Comment:
I suppose it's possible to optimize this a little to `fmt.Sprintf("field_name=%v", req.GetFieldName())`. Is the current way how it's currently implemented?

Reply:
Yeah, this is how it’s currently generated. But I’m happy to change it. It won’t have a big effect on the diffs

Reply:
I'll leave that up to you :)

Reply:
I took your suggestion :)